### PR TITLE
Improved Code Snippets across Mobile SDK's docs

### PR DESCRIPTION
**Pull request by desusai7**
## What do these changes do?

> Improved Code Snippets across Mobile SDK's docs.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> Add a link to the Notion ticket.

### CODE REVIEW

File: docs/sources/sdks/rudderstack-unity-sdk.mdx [hunk 1]
```diff
@@ -58,8 +58,7 @@ RudderClient.SerializeSqlite();
 
 // Build your config
```
Comment (ameypv-rudder) [inline docs/sources/sdks/rudderstack-unity-sdk.mdx:59]:
```suggestion
// Building your config
```

File: docs/sources/sdks/rudderstack-flutter-sdk/flutter-v1.mdx [hunk 2]
```diff
@@ -67,7 +67,6 @@ import 'package:rudder_sdk_flutter/RudderLogger.dart';
 RudderLogger.init(RudderLogger.VERBOSE);
```
Comment (arnab-p) [inline docs/sources/sdks/rudderstack-flutter-sdk/flutter-v1.mdx:67]: @desusai7 We should not ask to initialize the SDK with VERBOSE logging

**Review by arnab-p — CHANGES_REQUESTED** [review]

File: docs/sources/sdks/rudderstack-flutter-sdk/flutter-v1.mdx (hunk 2 above)
Comment (ameypv-rudder) [inline docs/sources/sdks/rudderstack-flutter-sdk/flutter-v1.mdx:67]:
```suggestion
```

**Comment by ameypv-rudder** [comment]
As discussed, @desusai7 and team will verify all the code snippets across the SDK docs.

**Comment by pfd** [comment]
Will revisit in the new docs framework if necessary

**Pull request closed without merging**